modules/references/client/components/create-reference/Feedback.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export default function Feedback({ feedback, onChangeFeedback }) {
+  const { t } = useTranslation('references');
+
+  return (
+    <div className="panel panel-default">
+      <div className="panel-heading">
+        <h4 id="feedback-public-question">
+          {t('Would you like to describe something about them?')}
+        </h4>
+      </div>
+      <div
+        className="panel-body"
+        role="group"
+        aria-labelledby="feedback-public-question"
+      >
+        <label htmlFor="feedback-message" className="control-label">
+          {t('Public feedback')}
+        </label>
+        <textarea
+          className="form-control input-lg"
+          rows="7"
+          id="feedback-message"
+          onChange={event => onChangeFeedback(event.target.value)}
+          value={feedback}
+        ></textarea>
+        <span className="help-block">
+          {t('Were they were they great at cooking? or singing?')}
```

Comment:
I'm OK with the text on the page, but I don't consider myself an expert on this.
Just correct this repeated excerpt:
"Were they were they"

Reply:
Fixed in https://github.com/Trustroots/trustroots/pull/1413/commits/513bc81b096bf7d88c9d5975ddf30e7a4cfddec1